tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "tslint:recommended",
   "rules": {
+    "max-classes-per-file": false,
```

Comment:
Now this is annoying, maybe we should apply this rule for /exceptions only? Is there an easy way to do it, like 2 configs with the 2nd one inheriting the main one and adding just one extra rule?

Reply:
Just leave it in the global one. No straightforward way to do it now.